docs/root/intro/arch_overview/statistics.rst

```diff
@@ -4,23 +4,39 @@ Statistics
 ==========
 
 One of the primary goals of Envoy is to make the network understandable. Envoy emits a large number
-of statistics depending on how it is configured. Generally the statistics fall into two categories:
+of statistics depending on how it is configured. Generally the statistics fall into three categories:
 
 * **Downstream**: Downstream statistics relate to incoming connections/requests. They are emitted by
   listeners, the HTTP connection manager, the TCP proxy filter, etc.
 * **Upstream**: Upstream statistics relate to outgoing connections/requests. They are emitted by
   connection pools, the router filter, the TCP proxy filter, etc.
+* **Server**: Server statistics describe how the Envoy server instance is working. Statistics like
+  server uptime or amount of allocated memory are categorized here.
 
 A single proxy scenario typically involves both downstream and upstream statistics. The two types
 can be used to get a detailed picture of that particular network hop. Statistics from the entire
 mesh give a very detailed picture of each hop and overall network health. The statistics emitted are
 documented in detail in the operations guide.
 
-Envoy uses statsd as the statistics output format, though plugging in a different statistics sink
-would not be difficult. Both TCP and UDP statsd is supported. Internally, counters and gauges are
-batched and periodically flushed to improve performance. Histograms are written as they are
-received. Note: what were previously referred to as timers have become histograms as the only
-difference between the two representations was the units.
+In the v1 API, Envoy only supports statsd as the statistics output format. Both TCP and UDP statsd
+are supported. As of the v2 API, Envoy has the ability to support custom, pluggable sinks. A few
+standard sink implementations are included in Envoy. Some sinks also support emitting statistics
+with tags/dimensions.
+
+Within Envoy and throughout the documentation, statistics are identified by a canonical string
+representation. The dynamic portions of these strings are stripped to become tags. Users can
+configure this behavior via the Tag Specifier configuration.
+
+Those three types of values are emitted as statistics:
```

Comment:
Why the change on this line?  I thought your original wording here was perfect: "Envoy emits three types of values as statistics:"

Reply:
This was to prevent beginning of the paragraphs from duplicating "Envoy" word: 56e6d48511d3089285fbf59997f6c08c41b01827.

Since the former paragraph has been reworded at dd23f7b, I will revert this change.